cdci_data_analysis/analysis/parameters.py

```diff
@@ -188,6 +187,7 @@ def __init__(self,
         self._allowed_values = allowed_values
         self.name = name
         self.units = units
+        self.default_units = default_units
```

Comment:
could be probably private (start with `_`) like `_allowed_units`. And also it should be checked that default units are in allowed units. For sanity.

Reply:
same approach used for checking `units`